.github/workflows/blank.yml

```diff
@@ -0,0 +1,29 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
```

Comment:
I wouldn't restrict the action with master branch only.
This will provide an opportunity to fix problems even before the PR is created.

Reply:
How would one know then that there is a problem?

Reply:
I mean, let's run shellcheck on any branch, not just master. 
With current settings, we have to create "PR to master" to start checking.

Reply:
I get it, but if we run shellcheck on any branch, how would we know that it failed? By going into the Actions page and searching for our new branch? Why not via a PR then?

Reply:
1. I make changes.
2. I create PR.
3. I see an error in check, I fix it, I update PR
4. goto 3
So, user have to create PR to check his work. But may be you are right. User can install shellcheck locally.

![image](https://user-images.githubusercontent.com/1169976/113249435-56f55d00-92c7-11eb-8dd1-f80de80b9137.png)
Green marker tell us that we have passed the check.